letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -88,20 +82,32 @@ class ApacheConfigurator(augeas_configurator.AugeasConfigurator):
 
     description = "Apache Web Server - Alpha"
 
+    # Kept in same function to avoid multiple compilations of the regex
+
+    private_ips_regex = re.compile(
```

Comment:
Seems like a useful regex, maybe put it to `util.py` (possibly even in the core)?